test/modules.coffee

```diff
@@ -457,7 +457,7 @@ test "export class", ->
       baz: ->
         console.log 'hello, world!'"""
   output = toJS input
-  ok /^export (class foo|var foo = \(function)/.test toJS input
+  ok /^export (var foo = class foo|var foo = \(function)/.test toJS input
```

Comment:
Why are we outputting the class name twice, i.e. `foo = class foo`? We don’t do that with functions: `foo = ->` becomes `foo = function() {};`, not `foo = function foo() {};`